pkg/hostagent/hostagent.go

```diff
@@ -378,19 +378,24 @@ func (a *HostAgent) startHostAgentRoutines(ctx context.Context) error {
 	if err := a.waitForRequirements(ctx, "essential", a.essentialRequirements()); err != nil {
 		mErr = multierror.Append(mErr, err)
 	}
-	mounts, err := a.setupMounts(ctx)
-	if err != nil {
-		mErr = multierror.Append(mErr, err)
-	}
-	a.onClose = append(a.onClose, func() error {
-		var unmountMErr error
-		for _, m := range mounts {
-			if unmountErr := m.close(); unmountErr != nil {
-				unmountMErr = multierror.Append(unmountMErr, unmountErr)
-			}
+
+	if a.y.MountType == "sshfs" {
+		// Set up **host** mounts, if any
```

Comment:
?

Reply:
For virtfs, we are just adding the params on the host side. The actual mount is done on the guest side, instead.

Reply:
Moved the type selector into setupMounts, so that we get the progress output and the host path creation